### ISSUE TITLE
simple typo in twisted/internet/error.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2512]
|<img alt="thomasvs's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| thomasvs reported|
|-|-|
|Trac ID|trac#2512|
|Type|enhancement|
|Created|2007-03-08 22:38:16Z|

This typo has been around a long time.

Attaching patch.  I hope this is what was meant to be said.

Attachments:

* [process.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1f3/1f3324e94a180e0f70d8bb309a1404f240102131/9ea0ea20446ecb340c51da47d5a9dcce673ca7e5.patch) (451 bytes) - added by thomasvs on 2007-03-08 22:38:38Z - 
* [error.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1f3/1f3324e94a180e0f70d8bb309a1404f240102131/45f18c6ee2cea55c2c5bb3185f49a722ccedf2a1.diff) (518 bytes) - added by exarkun on 2008-06-16 18:31:43Z - change excited to exited, leave has alone

<details><summary>Searchable metadata</summary>

```
trac-id__2512 2512
type__enhancement enhancement
reporter__thomasvs thomasvs
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1173393496000000 1173393496000000
changetime__1214070052000000 1214070052000000
version__None None
owner__ 
cc__moonfallen
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|moonfallen commented|
|-|-|

Um, as long as you're in there, could you fix the *other* typo in that sentence too?  "exited".  :-)

**Comment by twisted-trac** [comment]
|<img alt="thomasvs's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|thomasvs commented|
|-|-|

Oh, well how am I supposed to know which direction the developers wanted to go with their docstring ? I was just excited to be able to contribute.

I am happy if this gets fixed either way to uphold the excellent quality image of Twisted! Would it be better to do the fixing work on a branch ?

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve commented|
|-|-|

You should probably add a unittest to verify that the word 'excited' is not in the doctring anymore. It's not in the twisted 'family spirit' policy, because it vehicles subversive ideas.

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|moonfallen set owner to moonfallen<br>moonfallen set status to `assigned`|
|-|-|

No unit test or branch is necessary.  I just thought it was strange that you fixed one typo and not the other. :-P  Technically we're not supposed to review changes we made ourselves, and I'd like to review and commit your change.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

_has_ looks correct to me in that context.  I fixed _excited_ in the attached patch, though.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_7"></a>|@therve set owner to @exarkun|
|-|-|

Please apply.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [23960]) Apply error.diff, fixing the docstring of ProcessExitedAlready

Author: exarkun
Reviewer: therve
Fixes: [#2512](https://github.com/twisted/twisted/issues/2512)

Reformat the docstring of `ProcessExitedAlready` and fix the spelling of
"exited".

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
